racket/collects/racket/hash.rkt

```diff
@@ -84,4 +90,6 @@
                         #:combine/key
                         (-> any/c any/c any/c any/c)]
                        #:rest (listof hash?)
-                       (and/c hash? immutable?))])
+                       (and/c hash? immutable?))]
+ [hash-filter-values (-> (hash? procedure?) hash?)]
```

Comment:
This doesn't look like a valid contract to me. There's an extra pair of parens?

Reply:
thank you, I'm not used to using contracts in my daily programming so I wasn't sure how to structure these. I've updated the file to remove the extra parens